wandb/sdk/wandb_settings.py

```diff
@@ -532,6 +532,15 @@ class Settings(BaseModel, validate_assignment=True):
     TODO: Not implemented in wandb-core.
     """
 
+    x_on_run_stop: Callable[[], None] | None = Field(
```

Comment:
let's discuss before we implement this?

Reply:
sure, I made this a draft for a reason :)